engine/src/test/java/org/camunda/bpm/engine/impl/batch/BatchInvocationsPerJobByBatchTypeConfigTest.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.bpm.engine.impl.batch;
+
+import ch.qos.logback.classic.Level;
+import org.camunda.bpm.engine.ProcessEngine;
+import org.camunda.bpm.engine.ProcessEngineConfiguration;
+import org.camunda.bpm.engine.batch.Batch;
+import org.camunda.bpm.engine.impl.cfg.ProcessEngineConfigurationImpl;
+import org.camunda.commons.testing.ProcessEngineLoggingRule;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.entry;
+
+public class BatchInvocationsPerJobByBatchTypeConfigTest {
+
+  protected static final String PROCESS_ENGINE_CONFIG =
+      "camunda.cfg.invocationsPerJobByBatchType.xml";
+
+  protected static final String CONFIG_LOGGER = "org.camunda.bpm.engine.cfg";
+
+  @Rule
+  public ProcessEngineLoggingRule loggingRule = new ProcessEngineLoggingRule()
+      .watch(CONFIG_LOGGER)
+      .level(Level.WARN);
+
+  protected ProcessEngine processEngine;
+  protected ProcessEngineConfigurationImpl engineConfiguration;
+
+  @Before
+  public void setup() {
+    processEngine = ProcessEngineConfiguration
+        .createProcessEngineConfigurationFromResource(PROCESS_ENGINE_CONFIG)
+        .buildProcessEngine();
+
+    engineConfiguration =
+        (ProcessEngineConfigurationImpl) processEngine.getProcessEngineConfiguration();
+  }
+
+  @After
+  public void teardown() {
+    processEngine.close();
+  }
```

Comment:
Can we use the `new ProcessEngineRule(PROCESS_ENGINE_CONFIG)` here? I have the impression that this could remove the initialization from this test class.

Reply:
I decided not to use it because the logging rule wouldn't work. The log is written on bootstrap level and can therefore not be captured if bootstrapping is not part of the test.

Reply:
That sounds reasonable 👍